chaos_genius/databases/models/kpi_model.py

```diff
@@ -98,6 +102,19 @@ def as_dict(self):
             "created_at": self.created_at,
         }
 
+
+    @classmethod
+    def get_multidim_status(cls, id):
+        kpi_info = cls.get_by_id(id)
+        if not kpi_info or not kpi_info.data_source:
+            raise ValueError(f"KPI ID {id} not found in KPI_DATA")
```

Comment:
This can be made an instance method. Instead of getting the instance first and then using it in a classmethod, this can directly work on an instance.

Reply:
makes sense, fix incoming ...